code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -475,8 +478,12 @@
 /mob/dead/new_player/proc/transfer_character()
 	. = new_character
 	if(.)
+		if (client)
+			if (client.personal_lobby_music)
+				client.personal_lobby_music.stop_playing_to(client)
+			else
+				SSmusic.login_music?.stop_playing_to(client)
```

Comment:
this should be a proc since it's a larger expression used in the code more than once (stop lobby music for client)

Reply:
Resolved, it was replaced by stop_lobby_music